src/js/dbus.js

```diff
@@ -527,10 +541,16 @@ function registerService(name, service) {
 exports.Define = function(type) {
   if (type === String) {
     return 's';
-  } else if (type === 'Number') {
-    return 'i';
+  } else if (type === Number) {
```

Comment:
maybe use map style?
```js
{
  Number: 'i',
  String: 's',
...
}
```

Reply:
I tried this, but JerryScript seems not support this way.